go/private/actions/archive.bzl

```diff
@@ -65,7 +65,7 @@ def emit_archive(go, source = None, _recompile_suffix = "", recompile_internal_d
     if nogo != None and nogo.executable != None:
         out_facts = go.declare_file(go, name = source.name, ext = pre_ext + ".facts")
         out_nogo_log = go.declare_file(go, name = source.name, ext = pre_ext + ".nogo.log")
```

Comment:
Do you think we can put both log and patch in the same directory? logs can be empty too

Reply:
Great idea, will give that a try.